Tasks/Kasinskiy/Task5/task5.py

```diff
@@ -0,0 +1,39 @@
+def is_prime(x):
+    if isinstance(x, int) and x > 1:
+        for i in range(2, int(x ** 0.5) + 1):
+            if x % i == 0:
+                return False
```

Comment:
will be much cooler if you check a rem of 2, and then skip all the even numbers in this cycle